packages/react-error-overlay/src/containers/StackFrame.js

```diff
@@ -54,43 +68,45 @@ class StackFrame extends Component {
     }));
   };
 
-  canOpenInEditor() {
+  getEndpointUrl() {
     if (!this.props.launchEditorEndpoint) {
-      return;
+      return null;
     }
     const { _originalFileName: sourceFileName } = this.props.frame;
     // Unknown file
     if (!sourceFileName) {
-      return false;
+      return null;
     }
     // e.g. "/path-to-my-app/webpack/bootstrap eaddeb46b67d75e4dfc1"
     const isInternalWebpackBootstrapCode =
       sourceFileName.trim().indexOf(' ') !== -1;
     if (isInternalWebpackBootstrapCode) {
-      return false;
+      return null;
     }
     // Code is in a real file
-    return true;
+    return this.props.launchEditorEndpoint;
```

Comment:
Let's add `|| null` here so that it falls back to explicit `null` for bad values.

Reply:
Why making the distinction between null and undefined?